js/MVP/views/moteinnkallelse/Moteinnkallelse.tsx

```diff
@@ -48,18 +50,18 @@ const title = (type) => {
   }
 };
 
-const Moteinnkallelse = () => {
-  const { narmestelederId } = useParams();
+const Moteinnkallelse = (): ReactElement => {
+  const { narmestelederId } = useParams<{ narmestelederId: string }>();
 
   const sykmeldt = useSykmeldte(narmestelederId);
   const brev = useBrev(sykmeldt.data?.fnr);
   const mutation = useMutateBrevLest();
 
   const brevHead = Array.isArray(brev.data) ? brev.data[0] : null;
-  const { tid, uuid, brevType, document, lestDato } = brevHead;
+  const { tid, uuid, brevType, document, lestDato, videoLink } = brevHead || {};
```

Comment:
Uthenting av brevHead bør kanskje flyttes ned, slik at man vet at man har data.

Reply:
`uuid` er i bruk i useEffect hook som må stå før vi sjekker brev for error. Eller misforsto jeg deg?

Reply:
Problemet er at her ligger fallback-verdier (null for brevHead) og {} for verdiene, fordi man bruker data før man har sjekket isError | isLoading | isSuccess. 

Hadde det gått an å flyttet mutate-kallet til DocumentContainer  ? Da har man alltid et dokument.